R/RichDEM/build_tarballs.jl

```diff
@@ -56,7 +56,7 @@ VERBOSE=ON cmake --build . --config Release --target install -- -j${nproc}
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-julia_versions = [v"1.6.3", v"1.8"]
+julia_versions = [v"1.6.3", v"1.7", v"1.8"]
```

Comment:
Might want to add also the future versions we're already supporting:
```suggestion
julia_versions = [v"1.6.3", v"1.7", v"1.8", v"1.9", v"1.10"]
```